ui/component/commentMenuList/index.js

```diff
@@ -4,20 +4,23 @@ import { doCommentPin, doCommentModAddDelegate } from 'redux/actions/comments';
 import { doOpenModal } from 'redux/actions/app';
 import { doClearPlayingUri } from 'redux/actions/content';
 import { doToast } from 'redux/actions/notifications';
-import { selectClaimIsMine, selectClaimForUri } from 'redux/selectors/claims';
+import { selectClaimIsMine, selectClaimForUri, selectTitleForUri } from 'redux/selectors/claims';
 import { selectActiveChannelClaim } from 'redux/selectors/app';
 import { selectModerationDelegatorsById } from 'redux/selectors/comments';
 import { selectPlayingUri } from 'redux/selectors/content';
+import { selectUserVerifiedEmail } from 'redux/selectors/user';
 import CommentMenuList from './view';
```

Comment:
On a mention's context-menu, it seems like I can block a mentioned channel that I own -- but ends up with a commentron error.

I think similar to comment context-menu, the block/mute shouldn't appear on channel that you own.

Reply:
Adjusted. That menu now gives you the option to switch the active channel instead.